x-pack/plugins/security_solution/public/management/pages/host_isolation_exceptions/view/host_isolation_exceptions_list.tsx

```diff
@@ -60,6 +60,7 @@ export const HostIsolationExceptionsList = () => {
   const history = useHistory();
   const privileges = useEndpointPrivileges();
   const showFlyout = privileges.canIsolateHost && !!location.show;
+  const hasDataToShow = !isLoading && (!!location.filter || listItems.length > 0);
```

Comment:
This should be wrapped in a `useMemo`

Reply:
Maybe, but this calculation is order 1 and useMemo won't bring any performance improvement.

Reply:
Indeed. That makes sense. 👍